app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,163 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@DisplayName("TaskService 클래스")
+class TaskServiceTest {
+    abstract class Context_didCreateTwoTasks {
+        @BeforeEach
+        void context() {
+            final Task task1 = new Task();
+            task1.setTitle(FIXTURE_TITLE + 1);
+            final Task task2 = new Task();
+            task2.setTitle(FIXTURE_TITLE + 2);
+            service.createTask(task1);
+            service.createTask(task2);
+        }
+    }
+
+    public static final String FIXTURE_TITLE = "title";
+    private TaskService service;
+
+    @BeforeEach
+    void setup() {
+        service = new TaskService();
+    }
+
+    @Nested
+    @DisplayName("getTasks 메소드는")
+    class Describe_getTasks {
+        @Nested
+        @DisplayName("할일이 생성되지 않았을 때")
```

Comment:
이렇게 해보면 어떨까요?
```suggestion
        @DisplayName("생성되어 있는 할 일이 없다면")
```

D-C-I 스타일에서는 Describe, Context, It을 이어서 읽었을 때 하나의 문장이 되도록 해주면 좋습니다.
다음 두 문장을 비교하는 것처럼 대안이 될 설명을 생각하는 훈련을 해보세요.

- `getTasks 메소드는, 할 일이 생성되지 않았을 때, 빈 목록을 반환한다.`
- `getTasks 메소드는, 생성되어 있는 할 일이 없다면, 빈 목록을 반환한다.`

Reply:
제안 주신 구문이 좀 더 자연스럽네요
다들 부분들도 몇 번 더 읽어보겠습니다